CHANGES

```diff
@@ -18,12 +18,18 @@ The rules for this file:
   * 0.5.0
 
 Enhancements
+  - Visualisation module will change the data according to input unit (PR
```

Comment:
Reference #125 (generally, the issue is more important than the PR, you can include both if you like, but definitely reference the issue)